test/e2e/framework/framework.go

```diff
@@ -520,11 +520,6 @@ func (f *Framework) WaitForPodNotFound(podName string, timeout time.Duration) er
 	return e2epod.WaitForPodNotFoundInNamespace(f.ClientSet, podName, f.Namespace.Name, timeout)
```

Comment:
Can you more about why we are getting rid of the `WaitForPodReady` wrapper but not, for example, the `WaitForPodNotFound` wrapper?

Reply:
thanks for  your review . WaitForPod*()  in test/e2e/framework/framework.go  should be replaced to remove the dependency of core framework for sub pod . see  issue https://github.com/kubernetes/kubernetes/issues/81245